lib/hammer_cli_foreman/ssh.rb

```diff
@@ -12,6 +12,7 @@ class Command < HammerCLIForeman::Command
 
     command_name 'SSH to hosts'
     option %w(-c --command), 'COMMAND', _('Command to execute'), :attribute_name => :command, :required => true
+    option %w(-n --concurrent), 'CONCURRENCY', _('Number of concurrent SSH sessions'), :attribute_name => :concurrent
```

Comment:
You can add
```ruby
option ....... do |o|
  Integer(o)
end
```
for Hammer to ensure valid Integer is given

+1 to not force user to set concurrency limit